### PR TITLE
adding wasting category stratified mortality rate eqns

### PR DESCRIPTION
**Pull request by beatrixkh**

### CODE REVIEW

File: docs/source/gbd2019_models/risk_exposures/wasting_state_exposure/index.rst [hunk 1]
```diff
@@ -589,9 +589,50 @@ for now, this unknown is :math:`dur\_cat3`, which we will assume to be :math:`1/
 find values from the literature with which to update this. Note that the below
 equations also contain unknowns for :math:`d_1, d_2, d_3, d_4, f_1, f_2, f_3, f_4, p_0`.
 These are calculable from GBD data, but as these vary per age/sex category, we 
-leave them as variables here.
+leave them as variables here. Below we include equations for how these are calculated, and a link to a notebook implementing these equations.
 
-.. todo:: fill in the :math:`f_i` and :math:`d_i` vars with values/eqns
+.. math::
+
+   d_{cat 1&2}: acmr + sum([(emr_i*prev_i - cmsr_i) for i in diar,lri,msl]) + (emr_pem*1 - cmsr_pem)
```
Comment (rmudambi) [inline docs/source/gbd2019_models/risk_exposures/wasting_state_exposure/index.rst:596]: Doesn't this imply that the only difference in mortality between cats1&2 and cats3&4 is that cats1&2 have PEM? Shouldn't there be some reference to the RRs for each of these diseases?

Comment (Ninicorn) [reply docs/source/gbd2019_models/risk_exposures/wasting_state_exposure/index.rst:596]: The RRs of the disease determine the prevalence of the disease in the category. The CSMRs of the diseases are the same no matter what category you are in. prev_i is category specific.

**Review by Ninicorn — APPROVED** [review]
We need tp update the RR table to 2020 RRs and we need to figure out a way to make the transitions out of cat1 and 2 non-equal.

**Review by rmudambi — APPROVED** [review]

**Comment by aflaxman** [comment]
I know there are some changes still in the works for this, but I just read it and I think we might need to calculate the d_1,2,3,4 values slightly differently.  For example, d4 should be less than acmr - PEM_excess_mortality, because we know someone is CAT4 is less likely to have diarrhea, lri, or measles than someone in the general population, right?

**Comment by beatrixkh** [comment]
@aflaxman oh shoot; that was a typo, thank you! it was meant to be this: 

`    mortality_{cat 1&2}: acmr + sum([(emr_i*prev_i - cmsr_i) for i in diar,lri,msl]) + (emr_pem*1 - cmsr_pem)
    mortality_{cat 3&4}: acmr + sum([(emr_i*prev_i - cmsr_i) for i in diar,lri,msl]) - (cmsr_pem)`

**Comment by beatrixkh** [comment]
I'll open a new PR

**Pull request merged**